tests/Games/NexusMods.Games.TestFramework/AIsolatedGameTest.cs

```diff
@@ -336,6 +336,11 @@ private void ResetGameFolders()
     /// </summary>
     protected async Task<Loadout.ReadOnly> CreateLoadout()
     {
+        if (!_gameFilesWritten)
+        {
+            await GenerateGameFiles();
+            _gameFilesWritten = true;
+        }
```

Comment:
And this has no chance of creating parallel execution problems right? The files are generated in a different folder for each test?

Reply:
The Game Installation and DataModel location are randomised per run, and those places are obtained from DI, which is unique per run.

So I believe it's unique per run, given that the sources are.

Reply:
I'm not sure if Isolated tests were supposed to use inMemoryFS, could that be the case or did I dream that up?